pgraph/pgraph_test.go

```diff
@@ -323,34 +323,6 @@ func TestDeleteVertex2(t *testing.T) {
 	}
 }
 
-func TestVertexContains1(t *testing.T) {
-	v1 := NV("v1")
-	v2 := NV("v2")
-	v3 := NV("v3")
-	if VertexContains(v1, []Vertex{v1, v2, v3}) != true {
```

Comment:
You can leave all the tests in, let's just test the new methods you're using!

Reply:
Is there much point in testing stdlib functionality? The point of this change was to reduce the amount of code we're carrying. Less code -> easier to understand/reason about